content/docs/introducing-jsx.md

```diff
@@ -87,37 +96,38 @@ function getGreeting(user) {
 }
 ```
 
-### Specifying Attributes with JSX {#specifying-attributes-with-jsx}
+### Attributen specificeren met JSX {#specifying-attributes-with-jsx}
```

Comment:
```suggestion
### Attributen Specificeren met JSX {#specifying-attributes-with-jsx}
```
Although we haven't established any rules for this, significant words in titles tend to be capitalized.